compiler/lib/src/main/scala/syntax/Parser.scala

```diff
@@ -471,6 +471,7 @@ object Parser extends Parsers {
       constant ^^ { case _ => Ast.SpecLoc.Constant } |
       instance ^^ { case _ => Ast.SpecLoc.ComponentInstance } |
       port ^^ { case _ => Ast.SpecLoc.Port } |
+      state ~ machine ^^ { case _ => Ast.SpecLoc.StateMachine } |
```

Comment:
Delete this. It's not a correct lexical element.

Reply:
Fixed

Reply:
Actually this is needed for the fpp-depend tool.  I'm putting it back in.

Reply:
I agree, it's needed. I got confused between the lexer and the parser. Ignore my comment.